spec/grover/processor_spec.rb

```diff
@@ -95,6 +101,23 @@
         end
       end
 
+      context 'when puppeteer-core package only is installed', :remote_browser do
+        before do
+          FileUtils.move 'node_modules/puppeteer', 'node_modules/puppeteer_temp'
+        end
+        after do
+          FileUtils.move 'node_modules/puppeteer_temp', 'node_modules/puppeteer'
+        end
```

Comment:
Rubocop is either going to want a newline between these, or use the inline block form:
```suggestion
        before { FileUtils.move 'node_modules/puppeteer', 'node_modules/puppeteer_temp' }
        after { FileUtils.move 'node_modules/puppeteer_temp', 'node_modules/puppeteer' }
```